sklearn/tree/tests/test_export.py

```diff
@@ -378,11 +397,21 @@ def test_export_text():
     """
     ).lstrip()
 
-    assert export_text(clf) == expected_report
+    assert export_text(clf, class_names=None) == expected_report
     # testing that leaves at level 1 are not truncated
-    assert export_text(clf, max_depth=0) == expected_report
+    assert export_text(clf, class_names=None, max_depth=0) == expected_report
     # testing that the rest of the tree is truncated
-    assert export_text(clf, max_depth=10) == expected_report
+    assert export_text(clf, class_names=None, max_depth=10) == expected_report
```

Comment:
```suggestion
    assert export_text(clf, max_depth=0) == expected_report
    # testing that the rest of the tree is truncated
    assert export_text(clf, max_depth=10) == expected_report
```

Basically, it is this test that indicated me that we were doing wrong since `class_names=None` seems to be the previous behaviour.